BUILD.md

```diff
@@ -0,0 +1,90 @@
+# Building the Website
+
+This website is built on Middleman, a static site generator. See below for
+quick instructions:
+
+To get started, you need to have Ruby and Ruby Gems installed, as well
+as "bundler".
+
+
+## Initial setup
+
+On an rpm based distribution:
```

Comment:
On Fedora. I'm not sure this works on CentOS, and I'm really not sure about SUSE, etc.